migrations/20160214174319-create-link-to-shorten.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+module.exports = {
+  up: function(queryInterface, Sequelize) {
+    return queryInterface.createTable('linkToShortens', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      link: {
+        type: Sequelize.STRING
+      },
+      short: {
+        type: Sequelize.STRING
+      },
+      count: {
+        type: Sequalize.INTEGER
```

Comment:
You'll want to change this to 'Sequelize', not 'Sequalize', or you will have issues running migrations.